### DISCUSSION TITLE
Modification Forms

### DISCUSSION THREAD

**Discussion by Foucdvd** [neos-sdi/adfsmfa#286] [Q&A] [answered]
Hi,

I use adfsmfa as second authentification after login/password.

I want to use only OTP.

Is it possible to modify the text of forms ?

Example :
Current text (in French)
Scannez le QR Code avec une application TOTP tel que Microsoft Authenticator, Google Authenticator afin d'enregistrer votre compte

New Text:  (in french)
Si vous n'avez pas d'application TOTP pour scanner le QR CODE, vous pouvez utiliser une application TOTP tel que Microsoft Authenticator, Google Authenticator afin d'enregistrer votre compte. 

Thanks

**Comment by redhook62** [comment, marked as answer]
No, 

If you want to customize the user interface, you can take inspiration from the extension example provided, of course you will have to use Visual Studio and modify the resource files

regards
